src/components/ButtonGroup/ButtonGroup.jsx

```diff
@@ -88,16 +95,23 @@ ButtonGroup.defaultProps = {
   disabled: false,
   size: ButtonGroup.sizes.SMALL,
   kind: ButtonGroup.kinds.SECONDARY,
-  groupAriaLabel: ""
+  groupAriaLabel: "",
+  tooltipContainerSelector: PropTypes.string
```

Comment:
please change - `PropTypes.string` it is not a default value
please add default values for all other values